linkcheck/htmlutil/linkparse.py

```diff
@@ -251,3 +212,14 @@ def found_url(self, url, name, base, lineno, column):
         """Add newly found URL to queue."""
         assert isinstance(url, str_text) or url is None, repr(url)
         self.callback(url, line=lineno, column=column, name=name, base=base)
+
+def find_links(soup, callback, tags):
```

Comment:
PEP-8 asks for two blank lines between top level classes and functions.

```suggestion

def find_links(soup, callback, tags):
```

Reply:
Oops. Concentrated on `parser/__init__.py` and not that one. I will fix.